cmd/partial.go

```diff
@@ -44,6 +44,10 @@ func partial(cmd *cobra.Command, args []string) {
 	client := client.New(tok)
 	client.SetDryRun(dryrun)
 	client.SetSkipChannels(skipChannels)
+	
+	if dryrun {
+		log.Infof("This is a dry run. No messages will be deleted.")
```

Comment:
Log messages should be single sentences without a period at the end. But other than that, this change is good.
Sorry for the pedantry.

Reply:
Pedantry is important! I've now updated the message to say `No messages will be deleted in dry-run mode`